footprints/main/migrations/0039_auto_20200418_2007.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.12 on 2020-04-19 00:07
+
+from django.db import migrations
+
+
+def display_title(city, country):
+    parts = []
+    if city:
+        parts.append(city)
+    if country:
+        parts.append(country)
+
+    return ', '.join(parts)
+
+
+def migrate_city_country(apps, schema_editor):
+    Place = apps.get_model('main', 'Place')
+    for place in Place.objects.all():
+        title = display_title(place.city, place.country)
+        print(title)
```

Comment:
You can probably remove this `print()` statement. Just for debugging.

Reply:
Yes, it was. Thanks & resolved.